src/python/pants/backend/python/util_rules/python_sources.py

```diff
@@ -59,7 +59,7 @@ def __init__(
         targets: Iterable[Target],
         *,
         include_resources: bool = True,
-        include_files: bool = False
+        include_files: bool = True
```

Comment:
Just confirming that our deprecation policy still does not apply to rules. In options land, IIUC this would be verboten.

Reply:
Yep. We try to make a best-effort to minimize disruptions and document changes at https://www.pantsbuild.org/docs/plugin-upgrade-guide, but we have a huge warning that it's not stable at https://www.pantsbuild.org/docs/plugins-overview. 

Also, this type isn't documented on the website so we don't take as much care.

Reply:
OK. There were alot of "we"s there that I'm pretty sure are just you, but good to know and spread the soft rules. Thanks.